lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js

```diff
@@ -0,0 +1,10 @@
+/// <reference types = "Cypress"/>
```

Comment:
:+1: but maybe globally ? 

https://docs.cypress.io/guides/tooling/intelligent-code-completion.html#Reference-type-declarations-via-jsconfig

Reply:
@shilpabasu22 Hi Shilpa can you advise? This is based on your style.

Reply:
again we can implement this. Again this is not a blocker for your ticket and I can do this as part of the change that I will do for command.js file

Reply:
Sounds good and sensible. @a6software You happy with this?